### ISSUE TITLE
server: "unknown response ID" with loadbot.

### ISSUE THREAD

**Issue by JoeGruffins** [decred/dcrdex#2663]
Hit this while testing #2655 so it may not be on master. I originally thought that I saw the reason but I am unsure now. Will close this issue if it is not in master.

Client:
```
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: Received preimage request for order c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5 with known commitment 06db702b323d080f4ff86599c0570bca8a3f1c3d6e301d510cbd02a9258db2c3
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: Received preimage request for order 3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4 with known commitment 38139b325b75bd7ac933b0e002fa79be32ac7e39a0214b81280c550d88b76187
2024-01-08 17:30:41.507 [TRC] MANTLE:STACKER:0: book note received
2024-01-08 17:30:41.507 [INF] CORE:STACKER:1: Audited contract (dcr: d158dd1307f4ff0253a9e5e9dd7c309224868dd19830268b05b8e93b02ac7de8:0) paying to SsYCsUt7reFqrnzUzvv2J5b3jPXWi35tYhQ for order 17d2e6dcd1a3976af7524c80d8a7f2d8d0ee499de06930d86db260e6afcd8b05, match cc52bbeafdeae820ee02da0140a47fabedbbcdc8be26c29ff4527ce0a4d21c44, with tx data = true. Script: 6382012088c02067cf4b6f15dbe219fa7dceb06522332d6b3d05a082d4790f80f950fd107ec1f68876a9142a49cd7e2ee6f600e226070021019a7bbe5294c26704f0b39b65b17576a91468f860b6d40370ea684980cb76f9defa9fbd32f86888ac
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: Received preimage request for order 5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410 with known commitment f9a3b8e96c201523b9becb7aaf06995bffb793a93467f6f03c44336b84a12cf8
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: async processPreimageRequest for 3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4 succeeded
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: async processPreimageRequest for c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5 succeeded
2024-01-08 17:30:41.507 [DBG] CORE:STACKER:0: async processPreimageRequest for 5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410 succeeded
2024-01-08 17:30:41.507 [ERR] CORE:STACKER:0[wss://127.0.0.1:17273/ws]: No handler found for response: {"type":2,"id":1349,"payload":{"result":null,"error":{"code":32,"message":"unknown response ID"}}}
2024-01-08 17:30:41.507 [ERR] CORE:STACKER:0[wss://127.0.0.1:17273/ws]: No handler found for response: {"type":2,"id":1348,"payload":{"result":null,"error":{"code":32,"message":"unknown response ID"}}}
2024-01-08 17:30:41.507 [ERR] CORE:STACKER:0[wss://127.0.0.1:17273/ws]: No handler found for response: {"type":2,"id":1350,"payload":{"result":null,"error":{"code":32,"message":"unknown response ID"}}}
2024-01-08 17:30:41.507 [TRC] CORE:STACKER:0: Received a courtesy redemption request for match 937a4736f472bcdeacabd4073a8b3852775e35e070a2b420ecbdb95efa491696 as maker
2024-01-08 17:30:41.508 [DBG] CORE:STACKER:0[btc]: Rebroadcasting counterparty contract e09fc908aa4b1284dd6ef7b351b51e147667d71311b861848d1e200c5a76a1fc (THIS MAY BE NORMAL): rawrequest (sendrawtransaction) error: -27: Transaction already in block chain
```

And server:
```
2024-01-08 17:30:15.000 [TRC] MKT: Preimage request sent for order c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5
2024-01-08 17:30:15.000 [TRC] MKT: Preimage request sent for order 3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4
2024-01-08 17:30:15.000 [TRC] MKT: Preimage request sent for order 5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410
....
2024-01-08 17:30:35.001 [INF] MKT: No preimage received for order c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5 from user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f. Recording violation and revoking order.
2024-01-08 17:30:35.251 [DBG] AUTH: Recorded order 0c8c95d01fc83221f8d6283005415bd1c79e808389f7064d405db937be056888 that has finished processing: user=cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f, time=1704702635114, target=c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5
2024-01-08 17:30:35.251 [DBG] AUTH: RecordCancel: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 60, bond tier 100 => trading tier 100
2024-01-08 17:30:35.251 [DBG] AUTH: MissedPreimage: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 58, bond tier 100 => trading tier 100
2024-01-08 17:30:35.251 [INF] MKT: No preimage received for order 3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4 from user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f. Recording violation and revoking order.
2024-01-08 17:30:35.252 [INF] MKT: Sending a 'revoke_order' notification to cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f for order c2c8138ef1b88f0615d7c21a811eccb60108c91d8ae949c1dafd1cc37e9898b5
2024-01-08 17:30:35.538 [DBG] AUTH: Recorded order 409f99cbbbfe8aaed8ffa7694f93380814a8ad6bece268b5025a674b4751a2ca that has finished processing: user=cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f, time=1704702635457, target=3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4
2024-01-08 17:30:35.538 [DBG] AUTH: RecordCancel: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 58, bond tier 100 => trading tier 100
2024-01-08 17:30:35.538 [DBG] AUTH: MissedPreimage: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 56, bond tier 100 => trading tier 100
2024-01-08 17:30:35.538 [INF] MKT: No preimage received for order 5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410 from user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f. Recording violation and revoking order.
2024-01-08 17:30:35.538 [INF] MKT: Sending a 'revoke_order' notification to cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f for order 3a969070ea4d77f67b68a65b79084b252e2c28f5a4a40f71ce85d85a59269ae4
2024-01-08 17:30:35.817 [DBG] AUTH: Recorded order 8e2d7b9b40b3f99c1d68f36198006141fe6997c09a1001c1ec114e32e08da579 that has finished processing: user=cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f, time=1704702635680, target=5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410
2024-01-08 17:30:35.817 [DBG] AUTH: RecordCancel: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 56, bond tier 100 => trading tier 100
2024-01-08 17:30:35.817 [DBG] AUTH: MissedPreimage: user cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f strikes 54, bond tier 100 => trading tier 100
2024-01-08 17:30:35.817 [INF] MKT: Sending a 'revoke_order' notification to cf66dfacafa7479a9e5e80c70ad4129aaa667643aa23301d00360d3f104e757f for order 5701e2f5996266d3716b94dc6e2ef39d3836e4b5a72233ae796aa25b90e04410
```

**Comment by JoeGruffins** [comment]
Oh wait I misread, that is on error. hmmm.

**Comment by JoeGruffins** [comment]
Oh wow, maybe its my pc:
```
2024-01-08 17:30:39.986 [WRN] CORE:STACKER:1: checkTrades completed in 27.410330715s (slow)
2024-01-08 17:30:39.986 [WRN] CORE:STACKER:1: Long delay since previous trade check (just resumed?): 27.4103759s. Skipping this check to allow reconnect.
2024-01-08 17:30:39.986 [INF] LOADBOT[STACKER:1]: Balance: btc = 799001956 available, 140604334 locked
2024-01-08 17:30:40.360 [INF] CORE:STACKER:0: runJob(match) completed in 26.219820346s
```

Trade ticks are crazy.

**Comment by JoeGruffins** [comment]
Yeah I think my computer was just going slow for some reason. Request was sent after revoke. closing...
